examples/Cargo.toml

```diff
@@ -44,6 +44,10 @@ path = "css/main.rs"
 name = "custom_application"
 path = "custom_application/main.rs"
 
+[[bin]]
```

Comment:
It needs an entry on the README. Along with a readme/screenshot, like other examples.

Reply:
Yeah, especially for me, who doesn't really know GTK :) I have no idea what this example actually does